guide/popular-topics/reloading-commands.md

```diff
@@ -0,0 +1,58 @@
+# Reloading commands
+
+When writing your commands, you may find it tedious to restart your bot every time you want to test even your code's slightest change. However, a single bot command can reload commands if you have a command handler.
+
+This section assumes you did follow the [Command Handling](/guide/creating-your-bot/command-handling.md) part.
+
+```js {1}
+const { SlashCommandBuilder } = require('discord.js');
+
+module.exports = {
+	data: new SlashCommandBuilder()
+		.setName('reload')
+		.setDescription('Reloads a command.')
+		.addStringOption(option =>
+			option.setName('command-name')
+			.setDescription('Insert the command name.')
+			.setRequired(true)),
+	async execute(interaction) {
+		// ...
+	},
+};
+```
+
+In this command, you will be using a command name. First off, you need to check if the command you want to reload exists. You can do this check similarly to getting a command in your main file:
+
+```js {4-6,8-10}
+module.exports = {
+	// ...
+	async execute(interaction) {
+		const commandName = interaction.options.getString('command-name', true).toLowerCase();
+		const command = interaction.client.commands.get(commandName);
+
+		if (!command) {
+			return interaction.reply(`There is no command with name \`${commandName}\`!`);
+		}
+	},
+};
+```
+
+To build the correct file path, you will need the file name. You can use `command.data.name` for doing that.
```

Comment:
```suggestion
To build the correct file path, you will need the file name. You can use `command.data.name` for doing that.
```
this is incorrect if the user doesn't has file names same as the command names

Reply:
That's what the line 5 is for.

Reply:
the guide still recommends a /commands folder though so might be better to have all paths going there